Makefile

```diff
@@ -1,68 +1,75 @@
-# Set path so we can find poetry after install
-PATH := $(PATH):$(HOME)/.local/bin
+# Prepare environment for development
 
-deps:
+prepare:
 	sudo apt update
-	sudo apt install -y python3.9 python3.9-venv libpython3.9-dev libpq-dev graphicsmagick gcc python-dev
+	# Install python with virtual env and dev extensions, graphicsmagick, and gcc
+	sudo apt install -y python3.9 python3.9-venv libpython3.9-dev python-dev libpq-dev graphicsmagick gcc postgresql
 	# Install node: https://github.com/nodesource/distributions/blob/master/README.md#deb		
 	curl -fsSL https://deb.nodesource.com/setup_22.x | sudo -E bash -
 	sudo apt-get install -y nodejs
 	# Install poetry: https://python-poetry.org/docs/#installing-with-the-official-installer
 	curl -sSL https://install.python-poetry.org | python3.9 -
+	# Allow poetry to be called from the command line and make commands
+	@if ! echo "$$PATH" | grep -q "$$HOME/.local/bin"; then \
+		export PATH="$$PATH:$$HOME/.local/bin"; \
+	fi
```

Comment:
Ooo good call. I've updated it to add to `.bashrc` and to the current shell's path so we take care of current and future sessions

Reply:
I don't know if that will do it, since not everybody uses `bash`. I did recently add instructions to the README about setting this up, so it might be safe to simply remove from the Makefile

Reply:
Sorry, I missed this comment before pushing my latest version of the PR feedback commit. 

That's true, I liked the idea of automating it but maybe it isn't needed. I'll just have it print out a comment instead so a user sees it at the end of the `make prepare` command

Reply:
sgtm

Reply:
<img width="786" alt="Screenshot 2024-12-09 at 1 43 45 PM" src="https://github.com/user-attachments/assets/b4807af7-cd5c-45a1-acfa-cda73b92917a">

Thoughts?

Reply:
Looks good!